src/duns_schema.js

```diff
@@ -1,6 +1,9 @@
+var _ = require('underscore');
```

Comment:
This would be invalid to AirBnB style guide for es6. 

https://github.com/airbnb/javascript#10.1